### PR TITLE
Add additional viewport items that are coming in the 2205 release

### PR DESCRIPTION
**Pull request by hultonha**
Minor additions to call-out some upcoming viewport additions in the 2205 release

Signed-off-by: Tom Hulton-Harrop <82228511+hultonha@users.noreply.github.com>

### CODE REVIEW

**Review by Ulrick28 — APPROVED** [review]
Looks good, appreciate the callout the editor mode visual feedback is experimental.

**Comment by hultonha** [comment]
> Looks good, appreciate the callout the editor mode visual feedback is experimental.

Awesome thanks @Ulrick28 :+1: I believe I technically need one more approval before merging so I'll leave this for now (or as it's a doc change are you happy to hit the merge button?) Thanks!

**Review by tonybalandiuk — APPROVED** [review]
Looks good but I'd love to give readers a way to dig deeper if they are curious. See my comment. I approve of this and won't require the change, but if we have it, let's add it.

File: releases/22.05.0/22050 feature list.md
Comment (hultonha) [reply releases/22.05.0/22050 feature list.md]: Good call, I've updated the entry with links to the initial PR and roadmap item we have for the work

**Comment by hultonha** [comment]
> Looks good but I'd love to give readers a way to dig deeper if they are curious. See my comment. I approve of this and won't require the change, but if we have it, let's add it.

Thanks @tonybalandiuk! I added your suggestion, hopefully that's what you were after. Cheers!

**Comment by Ulrick28** [comment]
Go ahead and submit!

**Pull request merged**
